### ISSUE TITLE
Element inside an IF block make next blade fly away

### ISSUE THREAD

**Issue by CKylinMC** [kitschpatrol/svelte-tweakpane-ui#18]
When I use it in "if" block, it sometimes make the next blade fly away. I'm still finding what cause that. For reproduce, see [this](https://svelte.dev/repl/c6fe4cc94d894c90bac22d5bc7dc56a3?version=4.2.19):

To reproduce this, you need to put them into a tab, inside an IF block, and have blades and Element in the same time:

```
<TabGroup>
	<TabPage title="A">
		<Checkbox label="Check this" bind:value={check} />
		{#if check}
			<Text label="I should inside if block and before the element" bind:value={text} />
			<Element>
				Something here
			</Element>
			<Text label="I should inside if block and after the element" bind:value={text} />
		{/if}
		<Text label="outer text" bind:value={text} />
		<Text label="outer text" bind:value={text} />
	</TabPage>
</TabGroup>
```

and what I want:

![image](https://github.com/user-attachments/assets/dc35f406-bd5f-45e0-913a-fe8c290d611e)

what it actually goes:

![image](https://github.com/user-attachments/assets/8bda5e1f-eb8b-4bc8-8b7e-0e756e5aa2c4)

A text element inside IF block goes next to the outer text element.

**Comment by kitschpatrol** [comment]
Thank you very much for reporting this and for providing a clear and concise reproduction example.

I think I've fixed the issue, see the latest release [v1.3.3](https://github.com/kitschpatrol/svelte-tweakpane-ui/releases/tag/v1.3.3). (Feel free to re-open if you find otherwise.)

Maintaining consistent ordering of elements between Svelte / the DOM's declarative approach and Tweakpane's imperative approach has been one of the trickier parts of this project. 😅

**Comment by CKylinMC** [comment]
Thanks!
